.github/workflows/ci.yml

```diff
@@ -73,8 +73,9 @@ jobs:
         env:
           LSP: '${{ matrix.lsp }}'
           LSP_EXTRACT_FILE: '${{ matrix.lsp_extract_file }}'
-          # Should match 'name:' up above
-          JOB_NAME: 'Windows (${{ matrix.python }}, ${{ matrix.arch }}${{ matrix.extra_name }})'
+      - uses: codecov/codecov-action@v3
+        with:
+          directory: empty
```

Comment:
Could you add flags here? Look into sphinxcontrib-towncrier, cheroot, ansible/pylibssh and aiomysql for examples. Make sure the flag names don't have special symbols — they may break uploads.

Reply:
Surprisingly never heard of those, they sound neat! But... do we need them? The [docs](https://docs.codecov.com/docs/flags) say that they are for use with either different kinds of tests (which, meh, I don't think trio really shows?) or a monorepo.

Do you have a specific idea for how to use them here?

Reply:
Besides the test types (like with unit vs. integration running separately), I usually put the Python version and the OS there (hence the examples I mentioned). This allows seeing which code paths are tested under which runtimes.

Reply:
The UI also has a way of toggling the flags and you'll see which flags cover the lines or don't.